addons/product/migrations/13.0.1.2/pre-migration.py

```diff
@@ -174,7 +174,7 @@ def migrate(env, version):
     openupgrade.rename_xmlids(cr, _xmlid_renames)
     fill_product_pricelist_item_prices(env)
     fill_product_pricelist_item_active_default(env)
+    calculate_product_product_combination_indices(env)
     insert_missing_product_template_attribute_line(env)
     insert_missing_product_template_attribute_value(env)
-    calculate_product_product_combination_indices(env)
```

Comment:
If the ptvas were archived on v13, they has combination_indices, isn't it? Why not filling such ones then?

Reply:
Ok, you have a point here. I don't know why but I thought archived ptavs were avoided. Dropped that commit.